### ISSUE TITLE
Weird issue with goodies using share

### ISSUE THREAD

**Issue by killerfish** [duckduckgo/p5-app-duckpan#229]
**Issue:**
Goodies accessing the share directory fail to startup.
**How to reproduce:**
1. Create directory in the folder with your goodies repo named 't'
2. Fire up any goodie using share dir e.g. `duckpan server PeriodicTable`

I think another issue like this popped up a while back, but i don’t recall it precisely. @moollaza do you remember which issue was that?

**Comment by MrChrisW** [comment]
@killerfish Interesting, I've just fired up and tested however I'm unable to reproduce. Can you clarify what you mean when you say *"Create directory in the folder with your goodies repo named 't'"*
Where are you creating the directory named "t" ? in `/share/goodie/periodic_table` or elsewhere ?

**Comment by killerfish** [comment]
Sorry for the ambiguity, here is a SS
![weirdissue](https://cloud.githubusercontent.com/assets/1424236/6232689/89ebf2b0-b6ef-11e4-9b42-7104b313d408.png)
Im waiting on @moollaza since this might a be replicate of a similar issue (i forgot).

**Comment by MrChrisW** [comment]
@killerfish Looks like a local issue, I've tested again: 

![screenshot2135](https://cloud.githubusercontent.com/assets/5282264/6241710/d663199e-b75c-11e4-9e5b-be8ff0b47f26.png)

**Comment by killerfish** [comment]
@MrChrisW ill investigate this further. Im getting the same thing
![issuewithgoodies](https://cloud.githubusercontent.com/assets/1424236/6242520/9f221c46-b752-11e4-9029-1d28eb5804be.png)

**Comment by MrChrisW** [comment]
@killerfish Can you try updating your perl version? As per the duckpan message "v5.16.0> recommended" dobut it will resolve the issue but worth a try.

**Comment by killerfish** [comment]
@MrChrisW problem is with [this case] (https://github.com/duckduckgo/duckduckgo/blob/master/lib/DDG/Meta/ShareDir.pm#L88-L90), which is ancient, and needs to be removed, i think. Making a PR on duckduckgo for this. //cc @moollaza 
![oknow](https://cloud.githubusercontent.com/assets/1424236/6243820/98b09d56-b76b-11e4-866b-19edb99b6eec.png)

**Comment by MrChrisW** [comment]
@killerfish :+1: sure, still strange I'm unable to replicate.

**Comment by mintsoft** [comment]
@MrChrisW FWIW my ddh environment is `Perl 5.14.2` on Debian wheezy; never had a problem :+1:

**Comment by MrChrisW** [comment]
@mintsoft Cool, just trying to rule that out :smiley: 
@killerfish **Finally I was able to reproduce the issue!!!** That looks like the cause as it's checking the `t` dir above zeroclickinfo-goodies (root).

**Comment by moollaza** [comment]
Hey guys, thanks a lot for reporting this and thanks even more for making a PR @killerfish .

This is indeed a know bug, that crops up occasionally. I was about to say the problem exists somewhere in here: https://github.com/duckduckgo/duckduckgo/blob/master/lib/DDG/Meta/ShareDir.pm

As mentioned the problem occurs when a "/t" dir is a sibling to your repo dir AFAIK.
